zephyr/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 if(CONFIG_SOF)
```

Comment:
Please read this https://chris.beams.io/posts/git-commit/#seven-rules

Your commit message should provide more info to better understand WHY do we need it.!

Reply:
Some other very good ones:
https://wiki.openstack.org/wiki/GitCommitMessages
https://wiki.qemu.org/Contribute/SubmitAPatch (Ignore the QEMU-specific advice)
https://wiki.qemu.org/Contribute/SubmitAPullRequest (Ignore the QEMU-specific advice)